platform/ui/src/components/ViewportActionBar/ViewportActionBar.jsx

```diff
@@ -188,6 +198,7 @@ const ViewportActionBar = ({
       )}
       <div className="flex ml-4 mr-2" onClick={onPatientInfoClick}>
         <PatientInfo
+          showPatientInfoRef={showPatientInfoRef}
```

Comment:
It's a little weird that we have to prop-drill this ref. Can we wrap this component in a div or something else that we set `ref` on so we don't have to drill?

Reply:
Yes, I agree! I've removed the prop from tooltip and wrapped the component into a div 👍